src/isPalindrome.js

```diff
@@ -0,0 +1,8 @@
+export default function isPalindrome(string){
+  var newString = string.replace(/[^\w]_?/g,'').toLowerCase()
+  var lastString = newString.split('').reverse().join('')
+
+  return newString === lastString
+
+  //console.log(lastString)
+}
```

Comment:
Sweet this looks pretty good and simple. Maybe next time try it with a for loop. Just for fun.